modules/gltf/src/gltf/gltf-instantiator.ts

```diff
@@ -130,17 +130,19 @@ export class GLTFInstantiator {
   }
 
   createPrimitive(gltfPrimitive: any, i: number, gltfMesh): ModelNode {
+    const id = gltfPrimitive.name || `${gltfMesh.name || gltfMesh.id}-primitive-${i}`;
```

Comment:
Nit for other PR: It would be neat if this file was converted to a function, the same way I converted the PBR material code from a class to a function. I feel that having one-time use code packaged as a stateful class here just adds confusion.